### PR TITLE
#281 api syntax highlight

### LINKED ISSUE #281: [SYNC] - Simple Translation Full - M1
# Description

Add enough table and simple translations for them to satisfy M1 spec

### Scope

<ins>Includes </ins>
* item - (code)
* name - (code, is_customer, is_supplier)
* store - (code)
* ~~item_store_join~~
* ~~name_store_join~~ (would be part of remote sync, one way from central only)
* master_list_join
* master_list
* master_list_line

`list to be revised based on front end specs`

<ins>Excludes</ins>
* Api
* Complex translation in name_store_join

### Tasks

- [ ] Break down tasks into digestible (by reviewer) units.
- [ ] MW job added

### PR DESCRIPTION
**Pull request by andreievg**
Partial fixes
#281

### Changes

* Added prism, described in new doc info .md
* Tidied up a little bit to make sure syntax highlight works correctly

### CODE REVIEW

**Comment by andreievg** [comment]
Screen Shot 2021-09-05 at 1 09 02 AM

**Review by joshxg — APPROVED** [review]
Still some `transaction` needing to change to `invoice` at some point

File: docs/content/docs/api/patterns.md [hunk 1]
```diff
@@ -50,18 +50,18 @@ The sort object is two Key:Value pairs defining which field the result set is so
...
+   key: InvoiceSortField
    desc: Boolean
 }
 
-enum InvoiceSortOptions {
+enum InvoiceSortField {
   CODE
   OTHER_PARTY_NAME
-  ...
+  ...otherSortOptions
```
[the remark this answers is not in the document]
Comment (andreievg) [reply docs/content/docs/api/patterns.md:64]: Agree, would update in next PR

File: docs/content/docs/api/patterns.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ type IntFilter {
 type StringFilter {
   isNull: Boolean
   equalTo: String
-  ...
+  ...otherStringFilters
```
[the remark this answers is not in the document]
Comment (andreievg) [reply docs/content/docs/api/patterns.md:134]: Agree, would update in next PR

**Comment by andreievg** [comment]
> Still some transaction needing to change to invoice at some point

Yeah, i was going to make another PR with just the changes from latest translation decisions

**Comment by andreievg** [comment]
Also planning to change the api docs a bit once we have actual implementation of schema, using types from actual schema export: https://www.dropbox.com/s/c6qreaeiloz83pu/schema%20export.mov?dl=0

**Pull request merged**